tests/Froxlor/ValidateTest.php

```diff
@@ -109,13 +109,27 @@ public function testValidateCidrNoationToNetmaskNotationIPv4()
         $this->assertEquals("8.8.8.8/128.0.0.0", $result);
     }
 
+    public function testIPv6Checks()
```

Comment:
please stay conform with the naming regarding the rest of the tests, ergo `testValidateIPv6` would be more suitable

Reply:
Changed